packages/ckeditor5-editor-balloon/src/ballooneditoruiview.ts

```diff
@@ -20,6 +20,11 @@ export default class BalloonEditorUIView extends EditorUIView {
 	 */
 	public readonly editable: InlineEditableUIView;
 
+	/**
+	 * Menu bar view instance.
+	 */
+	declare public menuBarView: MenuBarView;
```

Comment:
Why did you use `declare`? We decided to do it using: `public override menuBarView: MenuBarView;`.